pkg/sr/api.go

```diff
@@ -348,42 +348,59 @@ func (cl *Client) Schemas(ctx context.Context, subject string) ([]SubjectSchema,
 	return schemas, firstErr
 }
 
-// CreateSchema attempts to create a schema in the given subject.
+// CreateSchemaID attempts to create a schema with a fixed ID and version ID in the given subject,
+// and returns the globally unique identifier assigned by the registry. If the id is set to -1 or 0,
+// the registry will try to allocate a new identifier or reuse an existing one (see below).
+// If the versionID is set to -1 or 0, it will be omitted when creating the schema.
 //
-// This supports param [Normalize].
-func (cl *Client) CreateSchema(ctx context.Context, subject string, s Schema) (SubjectSchema, error) {
-	return cl.CreateSchemaWithIDAndVersion(ctx, subject, s, -1, -1)
-}
-
-// CreateSchemaWithIDAndVersion attempts to create a schema with a fixed ID and
-// version ID in the given subject. If the id is set to -1 or 0, this method is
-// equivalent to CreateSchema(). If the versionID is set to -1 or 0, it will be
-// omitted when creating the schema.
+// If the same schema is registered under a different subject, the same identifier will be returned.
+// If the schema is identical to an existing schema in the registry, a new version will not be created.
+// Instead, the registry will return the identifier of the existing schema.
```

Comment:
I'd add some doc line like I mentioned below on create with id and version,

```I would add:
```
//
// If you want to register a schema and receive more details back,
// you can use [CreateSchema] or [CreateSchemaWithIDAndVersion]
// at the expense of another internal HTTP request.
```